package.json

```diff
@@ -6,8 +6,10 @@
     "@rails/activestorage": "^6.0.0-alpha",
     "@rails/ujs": "^6.0.0-alpha",
     "@rails/webpacker": "^5.3.0",
+    "glob-parent": "5.1.2",
```

Comment:
Do we want to use `^` on each to make it `^5.1.2`? I think this would allow other packages to start depending on newer versions one day.

Reply:
Good call 👍